gems/sorbet-runtime/test/types/edge_cases.rb

```diff
@@ -60,6 +60,17 @@ def ==(other)
     GC.stat[:total_allocated_objects] - before - 1 # Subtract one for the allocation by GC.stat itself
   end
 
+  private def assert_less_than(actual, expected, msg=nil)
```

Comment:
Minitest, in its Ruby cleverness, has an `assert_operator`, though I can't recall offhand if it accepts a message or not.

Reply:
Ah neat. Going to leave this one for now but good to know for the future.

Reply:
just kidding, the build failed, so I'll have to rebuild either way. updated now.